mm2src/mm2_main/src/lp_ordermatch.rs

```diff
@@ -5135,10 +5148,14 @@ pub async fn orders_kick_start(ctx: &MmArc) -> Result<HashSet<String>, String> {
 
     {
         let mut maker_orders_ctx = ordermatch_ctx.maker_orders_ctx.lock();
+        let mut orderbook = ordermatch_ctx.orderbook.lock();
         for order in saved_maker_orders {
             coins.insert(order.base.clone());
             coins.insert(order.rel.clone());
             maker_orders_ctx.add_order(ctx.weak(), order.clone(), None);
+            if let Some(key) = order.p2p_privkey {
+                orderbook.my_p2p_pubkeys.insert(hex::encode(key.public_slice()));
```

Comment:
It is not required here, as kick-started orders do not appear in the orderbook immediately. They require coins to be activated and have sufficient balance, which is checked in `lp_ordermatch_loop` later:
https://github.com/KomodoPlatform/atomicDEX-API/blob/6ede11586210d5fe641f0f1fb16844586b7a0c0f/mm2src/mm2_main/src/lp_ordermatch.rs#L3031-L3082

If all conditions are met, `maker_order_created_p2p_notify` is called - this will add new pubkey when needed.

Reply:
I see, thanks. Done.